ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -383,6 +389,7 @@ PrjFS_Result PrjFS_WritePlaceholderFile(
     
     if (nullptr == relativePath)
     {
+        LogWarning("PrjFS_WritePlaceholderFile: relative path is null %s ", relativePath);
```

Comment:
I think you might have meant to log the absolute path here, not the relative path which we've just established is null?

Reply:
I cleaned up the message here.  Unfortunately all we have here is the relative path.